README.md

```diff
@@ -1,2 +1,57 @@
-# split-expense
+# About
+
 A website to split expenses based on invoice
+
+# Local setup
+
+Application is built using python & flask-restplus as the backend and react for the front end. following are the steps required to setup the application in local computer
```

Comment:
Application is built using **[Python](https://www.python.org/)** and **[Flask-RESRPlus](https://flask-restplus.readthedocs.io/en/stable/)** as the backend and **[React JS](https://reactjs.org/)** as a front end technologies. Please follow the below steps for local application setup.

Reply:
Thanks for the review, Will update the requested changes :)